client/src/resources/roles.json

```diff
@@ -455,7 +455,7 @@
             {
                 "type": "playerRoleAndWill",
                 "role":"medium",
-                "will":"I always get medium and every game I get cleaned by a janitor, im calling it right now."
+                "will":"I always get medium and every game I get cremated by a mortician, im calling it right now."

```

Comment:
Wiki says they get told the role of the person they cremated. Where is that message?

Reply:
player role and will is the type